torchvision/models/shufflenetv2.py

```diff
@@ -51,6 +52,8 @@ def __init__(self, inp, oup, stride):
                 nn.BatchNorm2d(branch_features),
                 nn.ReLU(inplace=True),
             )
+        else:
+            self.branch1 = nn.Sequential()
```

Comment:
I'm merging this in, but it would be great if torchscript could support `Optional[nn.Module]`.